### PR TITLE
Configure build timeout in Jenkins

### PR DESCRIPTION
**Pull request by att55**
## Context
This PR implements that users can configure build timeout in Jenkins.

## Objective
Use the annotation for users to configure their own build timeouts in Jenkins.
As below...
```
annotations:
      beta.screwdriver.cd/executor: jenkins
      beta.screwdriver.cd/timeout: 30
```

## References
screwdriver-cd/screwdriver PR: https://github.com/screwdriver-cd/screwdriver/pull/1207
issue: https://github.com/screwdriver-cd/screwdriver/issues/1159

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

File: index.js [hunk 1]
```diff
@@ -230,6 +235,8 @@ class JenkinsExecutor extends Executor {
         this.username = options.jenkins.username || 'screwdriver';
         this[password] = options.jenkins.password;
         this.nodeLabel = options.jenkins.nodeLabel || 'screwdriver';
+        this.buildTimeout = options.jenkins.buildTimet || 90;
```
Comment (Filbird) [inline index.js]: buildTimeout*

**Review by Filbird — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (att55) [reply index.js]: Corrected 🙇

**Review by tkyi — APPROVED** [review]
LGTM 👍

**Review by jithine — APPROVED** [review]

**Pull request merged**